pkg/image/OWNERS

```diff
@@ -11,3 +11,4 @@ approvers:
   - soltysh
   - mfojtik
   - liggitt
+  - tnozicka
```

Comment:
Add yourself as a reviewer too

Reply:
I though approver is a superset includung reviewer. I think I've already done it somewhere and lgtm works there. What's the difference?

Reply:
now you will have to do image api reviews \o/ :-)

Reply:
oh no, I am going to fall back